File: app/src/main/java/com/karthik/myweather/network/model/Location.java
package com.karthik.myweather.network.model;

import com.google.gson.annotations.SerializedName;

import java.io.Serializable;

public class Location implements Serializable {

    @SerializedName("title")
    public String title;

    @SerializedName("woeid")
    public int locationId;
}

File: app/src/main/java/com/karthik/myweather/network/model/Location.kt
package com.karthik.myweather.network.model

import android.os.Parcelable
import com.google.gson.annotations.SerializedName
import kotlinx.android.parcel.Parcelize
import java.io.Serializable

@Parcelize
data class Location (
        @SerializedName("title")
        val title: String,

        @SerializedName("woeid")
        val locationId: Int) : Parcelable